CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -87,6 +92,33 @@ def run_query(year: int):
 
     conn = pyodbc.connect(connection_string)
 
+    #SQLite reports and cases tables setup
+    database_file_path = os.path.join(os.path.dirname(__file__), "database.db")
+    liteConn = sqlite3.connect(database_file_path) 
+    cur = liteConn.cursor()
+    #Reports table
+    cur.execute('''
+        CREATE TABLE IF NOT EXISTS Reports(
+            ID INTEGER PRIMARY KEY NOT NULL, 
+            createdAtDate TEXT,
+            timeOfCreation TEXT, 
+            numberOfDiscrepancies INTEGER    
+    )''')
+    #Cases table
+    cur.execute('''
+        CREATE TABLE IF NOT EXISTS Cases(
+            ID INTEGER PRIMARY KEY NOT NULL,
+            reportID INTEGER NOT NULL,
+            caseID INTEGER, 
+            eventCode TEXT,
+            MMWRYear INTEGER, 
+            MMWRWeek INTEGER,
+            reason TEXT, 
+            reasonID INTEGER,
+            FOREIGN KEY (reportID) REFERENCES Reports(ID)
+    )''')
```

Comment:
We will need to change these tables in the future depending on how our client wants the data structured. However, this is a great starting point that can be easily changed. Everything looks good to me!